subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -15,106 +15,204 @@
 [[gradle_wrapper]]
 == The Gradle Wrapper
 
-Most tools require installation on your computer before you can use them. If the installation is easy, you may think that’s fine. But it can be an unnecessary burden on the users of the build. Equally importantly, will the user install the right version of the tool for the build? What if they’re building an old version of the software?
+The recommended way to execute any Gradle build is with the help of the Gradle Wrapper (in short just “Wrapper”). The Wrapper is a feature that makes is extremely easy to execute a Gradle project without a Gradle installation. Upon invocation, it can _automatically download, unpack and install the Gradle runtime with a predefined version_ and use it to execute the build. As a result, developers can get up and running with a Gradle project quickly without having to follow manual installation processes saving your company time and money.
 
-The Gradle Wrapper (henceforth referred to as the “Wrapper”) solves both these problems and is the preferred way of starting a Gradle build.
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
This is really nice! Did it take very long to create?

Reply:
Maybe an hour or two. So not too long. I guess picking the icons really took the longest. ;-)

Reply:
D'oh! Kinda.